pkg/api/defaults-controller-manager.go

```diff
@@ -81,7 +82,10 @@ func (cs *ContainerService) setControllerManagerConfig() {
 	}
 
 	// Enables Node Exclusion from Services (toggled on agent nodes by the alpha.service-controller.kubernetes.io/exclude-balancer label).
-	addDefaultFeatureGates(o.KubernetesConfig.ControllerManagerConfig, o.OrchestratorVersion, "1.9.0", "ServiceNodeExclusion=true")
+	// ServiceNodeExclusion feature gate is GA in 1.19, removed in 1.22 (xref: https://github.com/kubernetes/kubernetes/pull/100776)
+	if !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.19.0") {
```

Comment:
nit: little awkward to read but looks like we don't have a LessThan function

Reply:
yeah this is the common pattern for less than

Reply:
/lgtm